bus-mapping/src/circuit_input_builder/input_state_ref.rs

```diff
@@ -313,6 +313,34 @@ impl<'a> CircuitInputStateRef<'a> {
         Ok(())
     }
 
+    /// Push a write type [`MemoryWordOp`] into the
```

Comment:
No need for a new function, the one above could just return the data.

Reply:
yes, use append-only for developing period. after all related pass, change old method itself. other it will cause many failures(relate to this method) at the change time.